esileapclient/osc/v1/node.py

```diff
@@ -48,20 +49,35 @@ def get_parser(self, prog_name):
             dest='lessee',
             required=False,
             help="Filter nodes by lessee.")
+        parser.add_argument(
+            '--property',
+            dest='properties',
+            required=False,
+            action='append',
+            help="Filter nodes by properties. Format: key>=value. \
+                Can be specified multiple times.",
```

Comment:
I'd add a note about having key/value set within quotes, due to the error you mentioned earlier